core-interfaces/package.json

```diff
@@ -1,16 +1,9 @@
 {
   "name": "@secret-agent/core-interfaces",
-  "version": "0.2.8",
+  "version": "1.0.0-alpha.0",
   "description": "Core interfaces used by SecretAgent",
-  "author": "Data Liberation Foundation",
-  "contributors": [
-    "Caleb Clark",
-    "Blake Byrnes"
-  ],
```

Comment:
Why cut out author and contributors?

Reply:
I changed it so we merge these all in during the build process so that we don't maintain in all these sub-packages.